sections/editing.include

```diff
@@ -184,6 +184,9 @@
   clicking it, the default action of the interaction event must be to <a>run synthetic click
   activation steps</a> on the element.
 
+  <p class="note">For accessibility, the keyboard's `Enter` and `Space` keys are often used to
+  trigger an elements [=activation behavior=]. [[wai-aria-practices-1.1]]</p>
```

Comment:
s/elements/element's/

`Enter` and in some cases `Space` - in particular, links generally don't activate with space, buttons generally do, and as noted elsewhere under `tabindex`, as far as I can find browsers only make this work by default on a pre-defined set of elements, so for example making something focusable with tabindex means that it will get click events but these won't be generated through the keyboard… :S